src/components/Map/GoogleMap.tsx

```diff
@@ -1,13 +1,13 @@
 import React, {useContext, useEffect, useMemo, useRef, useState} from 'react';
 
+import {Lang} from '@gravity-ui/uikit';
```

Comment:
U can use `'${Lang}'` type below (use backticks instead of single quotes). It is inconvenient to set such properties because of importing enums